h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -135,6 +139,15 @@ protected ModelMetricsListSchemaV3 schema(int version) {
             "specifying a Compact output format will produce a per-feature contribution", values = {"Original", "Compact"}, json = false)
     public Model.Contributions.ContributionsOutputFormat predict_contributions_output_format;
 
+    @API(help = "Predict the feature contributions - Shapley values - return top_n highest", json = false)
```

Comment:
it is not very clear these belong to "predict contributions"

Reply:
Reworded